source/_includes/common-tasks/backups.md

```diff
@@ -62,7 +62,7 @@ There are multiple ways to store the backup on another device:
 - **Option 4**: Or even better, create an automation to handle that, using one of the [backup actions](/integrations/hassio/#action-hassiobackup_full).
 - **Option 5**: Make use of a third-party add-on:
   - [Google Drive Backup](https://github.com/sabeechen/hassio-google-drive-backup)
-  - [Dropbox Sync](https://github.com/danielwelch/hassio-dropbox-sync)
+  - [Dropbox Backup](https://github.com/mikevansighem/dropbox-backup)
```

Comment:
This is replacing one non-working one with an archived one. I would suggest to remove it instead.

../Frenck

Reply:
That is also an option. Although the original link was useful for me in certain way.
We can also point to a fork, for example https://github.com/ianpleasance/dropbox-backup I have not tested that one.

Reply:
Let's just remove it in that case. :)

../Frenck

```suggestion
```